src/engine/shared/console.cpp

```diff
@@ -441,7 +441,10 @@ bool CConsole::ExecuteFile(const char *pFilename)
 	{
 		str_format(aBuf, sizeof(aBuf), "failed to open '%s'", pFilename);
 		Print(IConsole::OUTPUT_LEVEL_STANDARD, "console", aBuf);
-		Print(IConsole::OUTPUT_LEVEL_STANDARD, "console", "Info: only relative paths starting from the ones you specify in 'storage.cfg' are allowed");
+		if(pFilename[0] == '/' || pFilename[0] == '\\')
```

Comment:
Would add an additional check here if it starts with "x:\" for windows users, who probably wouldn't see that message otherwise

Reply:
Done.